duckduckgo/wireguard-apple
# Add iOS architectures and platforms

This PR adds iOS architectures and platforms to the XCFramework.

These changes are pretty ugly and could stand to be cleaned up, but it does the job for now.

---

Testing steps:

1. Check out this branch
2. Run `cd Sources/WireGuardKitGo`
3. Run `make build`
4. Wait for the framework to build
5. Once complete, check the contents of `Sources/WireGuardKitGo/out/WireGuard.xcframework`. You should see three entries:

```
ios-arm64
ios-arm64_x86_64-simulator
macos-arm64_x86_64
```
